packages/nimble-components/src/table/testing/table.pageobject.ts

```diff
@@ -186,6 +188,30 @@ export class TablePageObject<T extends TableRecord> {
         return anchor as Anchor;
     }
 
+    public getRenderedCellIcon(
+        rowIndex: number,
+        columnIndex: number
+    ): Icon | null {
+        const icon = this.getRenderedCellView(rowIndex, columnIndex).shadowRoot!
+            .firstElementChild;
+        if (!icon || !(icon instanceof Icon)) {
+            return null;
```

Comment:
Shouldn't we throw an error here? We do for other `getRenderedCell<>` variants (like `getRenderedCellAnchor`)?

Reply:
Sure, I just need to fix up the test to expect an error instead of `null`.